types/ckeditor__ckeditor5-core/src/editor/editorconfig.d.ts

```diff
@@ -60,7 +59,7 @@ type WordCountConfig = any;
 type TextPartLanguageOption = any;
 
 export interface EditorConfig {
-    alignment?: Alignment;
+    alignment?: AlignmentConfig;
```

Comment:
```suggestion
    alignment?: Alignment;
```
(depends on previous comment resolution)